CalculatorExercises/1-CalculatorBasic/Main.java

```diff
@@ -0,0 +1,27 @@
+import java.util.*;
+import java.lang.*;
```

Comment:
java.lang.* classes are imported automatically, you dont have to import it yourself.